src/map.c

```diff
@@ -32,6 +35,9 @@ void map_init(label_key_mode_e mode)
         label_keysyms = label_alpha_keysyms;
         map_length = LENGTH_ALPHA;
         break;
+    case LABEL_KEY_MODE_COLEMAK:
+        label_keysyms = label_colemak_keysyms;
+        map_length = LENGTH_COLEMAK;
```

Comment:
Please add a `break;` here, so we do not make mistakes when adding more key-mode options.

Reply:
ops, yeah ill fix it